### PR TITLE
Minor update of documentation for Translatable

### PR DESCRIPTION
**Pull request by svgrafov**
Update Translatable documentation to warn users of performance cap.
https://github.com/Atlantic18/DoctrineExtensions/issues/1502#issuecomment-171011176
https://github.com/Atlantic18/DoctrineExtensions/issues/860#issuecomment-26154908

### CODE REVIEW

**Comment by l3pp4rd** [comment]
well, there are many users who use it in production for larger projects. the point is that by default translatable casts foreign key to string (which is not optimal if you have integer). in that case optimization is to override the mapping of and change the field to integer.

It could mention instead that it may handle optimally up to million row tables with certain optimizations like using personal translations or overriding mapping of foreign key to prevent casting in database.

**Comment by TriAnMan** [comment]
@l3pp4rd not millions. 50k of translation rows is enough for performance cap mentioned in #1512

**Comment by l3pp4rd** [comment]
yes, but it mentions exactly that, you need to modify the type of **foreignKey** to improve the performance.

> Have fun with blazing speed of your DB queries. (In some cases we have accelerated our queries in more than 500 times: 130 milliseconds instead of 72 seconds)

So it is capable to handle big row size if the foreignKey is of correct type in order to prevent casting. That should mention it and point to that issue in order to update metadata for this column.

If we would say that it is capable to handle only small sets of data, that would not be **true**, because you can apply this optimization.

File: doc/translatable.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
 **Translatable** behavior offers a very handy solution for translating specific record fields
 in different languages. Further more, it loads the translations automatically for a locale
 currently used, which can be set to **Translatable Listener** on it`s initialization or later
-for other cases through the **Entity** itself
+for other cases through the **Entity** itself.
+
+Please note that **Translatable** behaviour is meant to be used for small projects and prototypes because of performance limitations.
```
Comment (l3pp4rd) [inline doc/translatable.md]: **NOTE:** that **Translatable** uses string type as foreignKey in order to be compatible with more types of columns.
When your database grows over 10K translated fields, if your **foreignKey** actually an **integer** for better performance update this column type, see #1512 for more details. If the performance gains are not well enough, consider a custom implementation for your translations.

**Comment by AkenRoberts** [comment]
Hello, and thank you for your contribution.

In the interest of spring cleaning and package modernization, I'm going through and closing old issues and pull requests that have had no recent activity, do not have associated tests, or have failing tests.

If you are still experiencing this issue or wish to contribute an up-to-date pull request, please create a new issue.

Thank you!

**Pull request closed without merging**